spyder/utils/tests/test_iofuncs.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+#
+
+"""
+Tests for iofuncs.py
+"""
+
+import os
+import pytest
+import numpy as np
+import spyder.utils.iofuncs as io
+
+__location__ = os.path.realpath(os.path.join(os.getcwd(),
+                                             os.path.dirname(__file__)))
+
+
+@pytest.fixture
+def real_values():
+    path = os.path.join(__location__, 'numpy_data.npz')
+    file_s = np.load(path)
+    A = file_s['A'].item()
+    B = file_s['B']
+    C = file_s['C']
+    D = file_s['D'].item()
+    E = file_s['E']
+    return {'A':A, 'B':B, 'C':C, 'D':D, 'E':E}
+
+def test_matlab_import(real_values):
```

Comment:
Please add a docstring to this test explaining how it works, i.e what are the contents of data.mat and numpy_data.npz, and what's the test expected result.